front/pages/api/w/[wId]/labs/transcripts/[tId].ts

```diff
@@ -128,6 +132,10 @@ async function handler(
         }
       }
 
+      if (dataSourceId !== undefined) {
```

Comment:
given the type check shouyld never be undefined no?

Reply:
Yeah it can - that patch endpoint is also used for setIsActive and the update happens depending on the param passed